src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
But isn't it easier to know something is wrong when `CallTarget` or `DependsOnTargets` fails because the target you asked for does not exist versus discovering we weren't running a tool because we renamed a target? Debugging is helpful when it fails with a noise, but target chaining fails pretty silently.